wolfcrypt/src/logging.c

```diff
@@ -41,13 +41,31 @@
     } 
 #endif
 
+#if defined(OPENSSL_EXTRA) || defined(DEBUG_WOLFSSL_VERBOSE)
```

Comment:
The null pointer for the queue is available in cases where openssl extra functions will be set to inspect it. The queue though is populated by calls to WOLFSSL_ERROR_LINE calling wc_AddErrorNode, which is surrounded by DEBUG_WOLFSSL.

Without DEBUG_WOLFSSL when wolfSSL code calls WOLFSSL_ERROR it will not be adding any nodes to the queue.

Without OPENSSL_EXTRA or DEBUG_WOLFSSL_VERBOSE there is no queue, did this to separate what was used in the compatibility layer. It could be changed to be part of default wolfSSL debug build though?